.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: node_js
 node_js:
-- "5"
-- "6"
-- "7"
 - "8"
```

Comment:
Dropping support for Node v5-v7, as they do not support most ES6 language features and have a status of"End of life"
https://github.com/nodejs/Release

If versions 5-7 need to be supported, I could look at adding Babel transpilation to the project to convert from ES6 to ES5.

Reply:
Dropping support for v5 is fine.  I think v6 & v7 support most of the ES6 language features though.  I'm thinking we should keep supporting those for now... everything else looks great though.  Thanks!

Reply:
Yes, versions 6 & 7 do support most ES6 language features. I imagine v6 will still have some active uses for awhile longer. I've adjusted the syntax to add back support for v6 and 7. 
We won't have the `...` spread operator, but most all the other language features remain.